api/sheets/get_variable.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Get Variable
+nav_order: 4
+parent: SHEETS
+grand_parent: API
+---
+
+# Get the value of a statistical variable at a given place and time.
+
+## =DCGET(dcids, variable, date)
+
+Given a list of [Place](https://browser.datacommons.org/kg?dcid=Place) DCIDs, a [StatisticalVariable](https://browser.datacommons.org/kg?dcid=StatisticalVariable), and a date, get the value of the variable at those places on that date.
```

Comment:
statistics values

Reply:
What do you mean by this?

Reply:
"get the value of the variable" is a bit mis-leading here? We want to say the stats(data) corresponding to the stats_var

Reply:
I changed it to "get the measurements of that variable at those places on that date." Sorry for the force push, I had to do it to fix the CLA problem that googlebot complained about.